### ISSUE TITLE
Alot of stages that use parallax get visual bugs

### ISSUE THREAD

**Issue by CheeksTheOriginal** [qingjun1991/GameDreamFactory#38]
here is an example of a stage: https://youtu.be/Y9tXZ9xLYYc?si=1VFbC6WjtpjALAcl

**Comment by qingjun1991** [comment]
Yes, because I don't have the source code for Mugen, I can't make it the same as Mugen. It can only be compatible with some regular stages and cannot display stages with special effects properly.

**Comment by CheeksTheOriginal** [comment]
> Yes, because I don't have the source code for Mugen, I can't make it the same as Mugen. It can only be compatible with some regular stages and cannot display stages with special effects properly.

Next update soon?

**Comment by qingjun1991** [comment]
No!

**Comment by NuggetFangus** [comment]
But will it someday be fixed? Also happy new year

**Comment by qingjun1991** [comment]
> But will it someday be fixed? Also happy new year

No!
happy new year too!
